server/models/Profile.js

```diff
@@ -0,0 +1,36 @@
+const mongoose = require("mongoose");
+const bcrypt = require("bcryptjs");
+
+const profileSchema = new mongoose.Schema({
+    first_name: {
+      type: String,
+    },
+    last_name: {
+      type: String,
+    },
+    gender:{
+        type: String,
+        type: String,
+    },
+    birth_date:{
+        type: String,
+    },
+    availability_dates:{},
+    email: {
+        type: String,
+     },
+    phone_number: {
+        type: Number,
+    },
+    where_you_live: {
+        type: String,
+    },
+    describe_yourself: {
+        type: String,
+    }
```

Comment:
camelCase is JavaScript convention and not snake_case, please going forward follow JavaScript convention. 👍🏻

Reply:
OK